Resources/Catalog/TSCatalogClient.py

```diff
@@ -16,15 +16,14 @@ class TSCatalogClient( FileCatalogClientBase ):
   """
 
   # List of common File Catalog methods implemented by this client
-  WRITE_METHODS = FileCatalogClientBase.WRITE_METHODS + [ "addFile", "removeFile" ]
+  WRITE_METHODS = FileCatalogClientBase.WRITE_METHODS + [ "addFile", "removeFile", "setMetadata" ]
+
+  NO_LFN_METHODS = [ "setMetadata" ]
```

Comment:
I admit I'm not sure about the definition of 'NO_LFN_METHODS', I've put it here in analogy with the FileCatalogClient.